cdap-app-fabric/src/main/java/io/cdap/cdap/internal/provision/DefaultSystemProvisionerContext.java

```diff
@@ -34,18 +34,22 @@
  */
 public class DefaultSystemProvisionerContext implements ProvisionerSystemContext {
 
+
   private final String prefix;
   private final CConfiguration cConf;
   private final AtomicReference<Map<String, String>> properties;
   private final String cdapVersion;
   private final Map<String, Lock> locks;
+  private final long reloadInterval;
+  private long lastReload = System.currentTimeMillis();
```

Comment:
Better to name the field as `lastConfReloadTime` and `confReloadInterval`

Reply:
Done